pkg/util/typeutil/schema_test.go

```diff
@@ -2057,7 +2057,7 @@ func TestValidateSparseFloatRows(t *testing.T) {
 			CreateSparseFloatRow([]uint32{100, 2000, 500}, []float32{1.0, 2.0, 3.0}),
 		}
 		err := ValidateSparseFloatRows(rows...)
-		assert.Error(t, err)
```

Comment:
this test is to catch invalid user input, not to test `CreateSparseFloatRow`

Reply:
change it back